Assets/Editor/ProdBuildScript.cs

```diff
@@ -15,6 +15,12 @@ public static void MyBuild()
         
         foreach (EditorBuildSettingsScene e in EditorBuildSettings.scenes)
         {
+            //Do not include AdditiveDebugScene in ProdBuild
+            if (e.path.Contains("AdditiveDebugScene"))
```

Comment:
Side thought. 
Debug scene will be the first additively added scene, and we're going to be adding more other ccomponents

Does it mean we'll be naming the other scenes Additive___Scene?

Reply:
That's what I was thinking for a naming convention for additive scenes. So that we distinctly know that these scenes will never be the primary scene (so people don't accidentally put something like a world manager or something like that into the scene).